app/services/applicants/create_student_service.rb

```diff
@@ -34,11 +34,13 @@ def create(tags)
 
     def create_new_student(tags)
       # Create a user and generate a login token.
+      user = school.users.with_email(@applicant.email).first
       user =
-        school
-          .users
-          .with_email(@applicant.email)
-          .first_or_create!(email: @applicant.email, title: "Student")
+        User.create(
+          school: school,
+          email: @applicant.email,
+          title: "Student"
+        ) if user.nil?
```

Comment:
I don't see why this change is necessary... it doesn't seem to do anything different from the earlier code... This looks like an expanded version of `first_or_create!`, but without the bang-version `create!` 😕.

Reply:
@harigopal Yes. I hadn't fully understood this either 😅 . But this was from a suggestion you made for fixing an error. See https://discord.com/channels/821286693512282142/1181611860588507187/1186717799641522216